core/src/main/java/org/elasticsearch/search/internal/ContextIndexSearcher.java

```diff
@@ -135,81 +77,19 @@ public Weight createNormalizedWeight(Query query, boolean needsScores) throws IO
         }
     }
 
-
-    @Override
-    public void search(Query query, Collector collector) throws IOException {
```

Comment:
also great to see this moved to QueryPhase, so this collector wrapping is only done at the place where it really required.